Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -292,14 +292,15 @@
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Nest Eggs in PlatinumQuest are generally much more difficult to find and collect over their Easter Egg counterparts in the other games.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: You can use a Super Speed to slow down, too. Just activate it while facing away from where you're going!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In PlatinumQuest, unlike other games, respawning on a Checkpoint will cancel a currently-active Time Travel's effects.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're new, the recommended order to play the games is: Gold, Ultra, PlatinumQuest, then Platinum.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you're new, the recommended order to play the games is: Ultra, Gold, PlatinumQuest, then Platinum.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Watching others play may give you tips on how to solve certain challenges.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: All but Orange-banded cannons allow you to aim them and fire at will, at varying power levels. Orange-banded cannons fire the marble instantly in the direction it's facing.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Fireball Powerup fizzles out when you touch water or collect a Bubble Powerup. Likewise, the Bubble Powerup pops away when you pick up a Fireball Powerup.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Bubble Powerup stays with you at all times until it is used up or you pick up a Fireball Powerup.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Sundial Item acts the same way as the Time Travel, it's just a bit fancier.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: PlatinumQuest's Time Travel Items have identifiable bands; gold bands freeze the clock and red bands add time.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Marble Blast Gold and Platinum's Time Travel Items can be identified by their sand color; yellow sands freeze the clock and red sands add time.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you activate a Mega Marble and hit an Ice Shard, your marble will shrink but remain unfrozen.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Colliding with an Ice Shard as a Mega Marble will shrink your marble to its regular size instead of freezing you.";
```

Comment:
What is the difference between these two?

Reply:
I might've meant to delete the old one? But also many other more generally relevant tips also have duplicates. I could get rid of the second one though

Reply:
Tbh I think the original one is worded better. Though, the newer one is better at explaining that you have to _activate_ mega marble, so it's clear this doesn't apply to levels where you're always mega (Unless it does? I really don't know)

Maybe: `Tip: Colliding with an Ice Shard while a Mega Marble is activated will shrink your marble to its regular size instead of freezing you.`

Reply:
Yeah idk why I worded it the way I did. I usually word stuff like "with an active Mega Marble" or "with an active Bubble" and such so idk what I was on with that